src/views/clusteroverview/SettingsTab/ClusterTab/components/AutomaticSubscriptionRHELGuests/utils/constants.ts

```diff
@@ -2,3 +2,5 @@ export const ACTIVATION_KEYS_URL = 'https://console.redhat.com/settings/connecto
 export const ACTIVATION_KEYS_DOCUMENTATION_URL =
   'https://access.redhat.com/documentation/en-us/red_hat_insights/2023/html/remote_host_configuration_and_management/activation-keys';
 export const REDHAT_CONSOLE_URL = 'https://console.redhat.com/';
+
+export const AUTOMATIC_UPDATE_FEATURE_NAME = 'auto-update-rhel-vms';
```

Comment:
Maybe it would be good to have the constants placed in this file alphabetically?

Reply:
:-(